package.json

```diff
@@ -49,7 +49,7 @@
     "babel-runtime": "~5.8.3"
   },
   "peerDependencies": {
-    "graphql": "~0.4.2"
+    "graphql": "~0.5.0"
```

Comment:
Looks like we need to change line 63 as well, if I'm reading the travis run correctly?